tests/boards/mec15xxevb_assy6853/qspi/src/main.c

```diff
@@ -446,7 +446,7 @@ void test_spi_quad_write(void)
 				"error %d", ret);
 
 		spi_status2 = safbuf2[0];
-		zassert_true((spi_status2 & SPI_STATUS2_QE) == 1,
+		zassert_true((spi_status2 & SPI_STATUS2_QE) == 2U,
```

Comment:
@chen-png  would the following have the same effect and be a bit less obscure 
zassert_true(spi_status2 & SPI_STATUS2_QE) == SPI_STATUS2_QE)

Reply:
yes, I have updated it, thanks.